modules/trusted-firmware-m/Kconfig.tfm

```
@@ -49,6 +49,13 @@ menuconfig BUILD_WITH_TFM
 
 if BUILD_WITH_TFM
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M output"
```

Maybe more descriptive:
```suggestion
	bool "Use UART1 for TF-M logging"
```